src/backend/joanie/signature/backends/dummy.py

```diff
@@ -98,3 +99,43 @@ def _send_email(self, recipient_email: str, reference_id: str):
             reference_id,
             recipient_email,
         )
+
+    def get_signed_file(self, reference_id: str) -> bytes:
+        """
+        Dummy method that returns a contract in PDF bytes if the reference exists.
+        """
+        if not reference_id.startswith(self.prefix_workflow):
+            raise ValidationError(
+                f"Cannot download contract with reference id : {reference_id}."
+            )
+        default_context = {
+            "contract": {
+                "body": "<p>Articles de la convention</p>",
+                "title": "CONTRACT DEFINITION 1",
+            },
+            "course": {
+                "name": "<COURSE NAME DEFAULT>",
+            },
+            "student": {
+                "name": "John Doe",
+                "address": {
+                    "address": "1 Rue de L'Exemple",
+                    "city": "Paris",
+                    "country": "FR",
+                    "first_name": "John",
+                    "last_name": "Doe",
+                    "is_main": True,
+                    "postcode": "75000",
+                    "title": "Office",
+                },
+            },
+            "organization": {
+                "logo": "organization_logo.png",
+                "name": "Organization 1",
+                "signature": "organization_signature.png",
+            },
+        }
```

Comment:
Through the reference id we should be able to retrieve the related contract then bind contract context nope ?